Sources/TuistCache/Cache/CacheLocalStorage.swift

```diff
@@ -38,49 +37,37 @@ public final class CacheLocalStorage: CacheStoring {
 
     // MARK: - CacheStoring
 
-    public func exists(name _: String, hash: String) -> Single<Bool> {
-        Single.create { completed -> Disposable in
-            completed(.success(self.lookupCompiledArtifact(directory: self.cacheDirectory.appending(component: hash)) != nil))
-            return Disposables.create()
-        }
+    public func exists(name _: String, hash: String) async throws -> Bool {
```

Comment:
Does this need to be defined `async throws` even if it is not to match protocol conformance, or it would conform anyway (I would expect so)?
Same for the other methods below

Reply:
Yep, it would conform anyway, I will change it ☺️